src/Angor/Shared/Services/MempoolSpaceIndexerApi.cs

```diff
@@ -115,7 +115,7 @@ private class Outspent
     public async Task<List<ProjectIndexerData>> GetProjectsAsync(int? offset, int limit)
     {
         var indexer = _networkService.GetPrimaryIndexer();
-        var response = await _httpClient.GetAsync($"{indexer.Url}{AngorApiRoute}/projects?offset={offset ?? 0}&limit={limit}");
+        var response = await _httpClient.GetAsync($"{indexer.Url}{AngorApiRoute}/projects?offset={offset ?? null}&limit={limit}");
```

Comment:
Null breaks this in the mempool side, also we don't need to default to null if it is null

Reply:
this is a problem then, how do we get the latest on the mempool indexer?

Reply:
fixed